library/src/main/java/com/android/volley/RequestQueue.java

```diff
@@ -343,6 +356,8 @@ void finish(Request<?> request) {
                 }
             }
         }
+
+        mRequestLogger.log(request);
```

Comment:
why not move this (and the `setRequestStartTime()`) into `Request.setRequestQueue()`? Seems like the correct place to be put